File: Clients/Harvey.Store.Web/src/app/shared/models/app-setting.model.ts
export class AppSettingModel {
  public id: string;
  public name: string;
  public value: string;
  public groupName: string;
  public appSettingTypeId: number;
}

export enum AppSettingType {
  adminApp = 'AdminApp',
  storeApp = 'StoreApp',
  memberApp = 'MemberApp',
}

export class GetAppsettingResponse {
  public appSettingModels: Array<AppSettingModel> = [];
}

File: Servers/Harvey.CRMLoyalty.Application/Models/AppSettingModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Harvey.CRMLoyalty.Application.Models
{
    public class AppSettingModel
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Value { get; set; }
        public string GroupName { get; set; }
        public int AppSettingTypeId { get; set; }
        public string AppSettingType { get; set; }
    }
}
